### PR TITLE
Updated DataSource and DataType according to new template

### PR DESCRIPTION
**Pull request by YaronFruchtmann**
Fixes #
Added DataSource

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficFromIPLeast.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Id: aa497951-c779-4ea2-be2a-127ea66c5fba
 // DisplayName: Hosts Receiving Least Amount of Data from IP
 // Description: Hosts receiving least Amount of data from IP during the range of -1d and +1d
-// DataSource: WireData
+// DataSource: ; DataTypes: #WireData
```
Comment (shainw) [inline Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficFromIPLeast.txt]: Confirming that DataSource is supposed to be empty?

File: Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficFromIPMost.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // Id: 73fb9b8d-fd13-4c43-8136-6d693cafaa23
 // DisplayName: Hosts Receiving Most Amount of Data from IP
 // Description: Hosts Receiving Most Amount of Data from IP during the range of -1d and +1d
-// DataSource: WireData
+// DataSource: ; DataTypes: #WireData
```
Comment (shainw) [inline Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficFromIPMost.txt]: Same here, DataSource empty?

File: Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficToIPLeast.txt [hunk 3]
```diff
@@ -6,7 +6,7 @@
 // OutputEntityTypes: [Host]
 // QueryPeriodBefore: 1d
 // QueryPeriodAfter: 1d
-// DataSource: WireData
+// DataSource: ; DataTypes: #WireData
```
Comment (shainw) [inline Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficToIPLeast.txt]: Same here, DataSource empty?

File: Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficToIPMost.txt [hunk 4]
```diff
@@ -6,7 +6,7 @@
 // OutputEntityTypes: [Host]
 // QueryPeriodBefore: 1d
 // QueryPeriodAfter: 1d
-// DataSource: WireData
+// DataSource: ; DataTypes: #WireData
```
Comment (shainw) [inline Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficToIPMost.txt]: Same here, DataSource empty?

**Comment by YaronFruchtmann** [comment]
Hey Shain,

Yes, WireData does not have a DataSource (a.k.a DataConnector), and it is suppose to be empty

File: Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficFromIPMost.txt (hunk 2 above)
Comment (YaronFruchtmann) [reply Exploration Queries/InputEntity_IP/IP2Host_HostByTrafficFromIPMost.txt]: yes, it is ok

**Pull request merged**